### DISCUSSION TITLE
Emoji favorites

### DISCUSSION THREAD

**Discussion by vVanile** [stoatchat/discussions#576] [Feature Suggestions]
The ability to favorite specific emotes and an option to only have favorited emoji show up in the suggestions when typing. or at least an option to turn off default emoji suggestions.

**Comment by void-stare** [comment]
Definitely need this. Many of us jumped ship from Discord but still miss some of the conveniences it has that Stoat doesn't yet; Favorite Emojis (and those being top of the emoji picker list) being one of them

**Comment by gabrielfordevelopment** [comment]
Agreed. Need an option to favorite emojis, and a "Recently used" emojis that show the last X emojis that were sent
